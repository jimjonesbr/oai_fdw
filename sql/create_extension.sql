CREATE EXTENSION oai_fdw;
CREATE SERVER oai_server FOREIGN DATA WRAPPER oai_fdw;
