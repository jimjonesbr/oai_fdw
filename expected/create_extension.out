CREATE EXTENSION oai_fdw;
-- No metadataPrefix
CREATE SERVER oai_server_err1 FOREIGN DATA WRAPPER oai_fdw 
OPTIONS (url 'https://services.dnb.de/oai/repository');
-- No url
CREATE SERVER oai_server_err2 FOREIGN DATA WRAPPER oai_fdw 
OPTIONS (metadataPrefix 'oai_dc');
-- No options 
CREATE SERVER oai_server_err3 FOREIGN DATA WRAPPER oai_fdw;
-- Warning: unknown option
CREATE SERVER oai_server_warn1 FOREIGN DATA WRAPPER oai_fdw 
OPTIONS (url 'https://services.dnb.de/oai/repository',
         metadataPrefix 'oai_dc',
         foo 'bar');         
        
CREATE SERVER oai_server_dnb FOREIGN DATA WRAPPER oai_fdw 
OPTIONS (url 'https://services.dnb.de/oai/repository',
         metadataPrefix 'oai_dc');
CREATE SERVER oai_server_ulb FOREIGN DATA WRAPPER oai_fdw 
OPTIONS (url 'https://sammlungen.ulb.uni-muenster.de/oai',
         metadataPrefix 'oai_dc');         
         
